mix.exs

```diff
@@ -122,7 +122,7 @@ defmodule Appsignal.Mixfile do
       {:plug_cowboy, "~> 1.0", only: [:test, :test_no_nif]},
       {:bypass, "~> 0.6.0", only: [:test, :test_no_nif]},
       {:ex_doc, "~> 0.12", only: :dev, runtime: false},
-      {:credo, "~> 1.0.0", only: [:test, :dev], runtime: false},
+      {:credo, "~> 1.5.0", only: [:test, :dev], runtime: false},
```

Comment:
```suggestion
      {:credo, "~> 1.5.6", only: [:test, :dev], runtime: false},
```

Should we lock it to the patch version you mention in the commit? This way we don't accidentally upgrade and some builds start to fail on new rules.